gk/main.c

```diff
@@ -1153,6 +1195,22 @@ cleanup_gk(struct gk_config *gk_conf)
                 destroy_mailbox(&gk_conf->instances[i].mb);
 	}
 
+	for (i = 0; i < (int)gk_conf->gk_max_num_ipv4_fib_entries; i++) {
```

Comment:
It's safer to define variable `unsigned int ui;` and use it here instead of using variable `i`. The reason for that is that `gk_max_num_ipv4_fib_entries` might be bigger than the biggest value a signed integer can hold.

The same goes for the `for` loop below.